src/App.tsx

```diff
@@ -1,23 +1,18 @@
 import './App.css';
+import { stockTicker$ } from './stockTicker.service';
+import { Tick } from './types/tick';
 import { useEffect } from 'react';
 
 function App() {
-
   useEffect(() => {
-    if(!window.StockTicker) {
```

Comment:
are we sure that we want to use RXJS in a react project? is that common at all?

Reply:
For the coding assessment, candidates don't need to worry about how the stream of stock ticks are generated. Using RxJS makes this React version more closely match the Angular version and I think it's a cleaner implementation.

Do you frequently have candidates study the implementation of the stock ticker stream? This hasn't come up so far in interviews I've conducted.

Reply:
I haven't, I've had one or 2 check it out from curiosity.

This is great, only request I would have is to make sure to update our `hello-janus-react` repository to have the same exact dependencies as this one now